suit_generator/cmd_image.py

```diff
@@ -325,7 +335,7 @@ def as_intelhex(self, storage_domain: ManifestDomain = None):
                 envelope_bytes = self._envelopes[role].ljust(max_size, b"\xFF")
                 envelope_count += 1
             else:
-                envelope_bytes = b"\xFF" * max_size
+                continue
```

Comment:
What's the case, in which this logic erased valid data?
The idea was that there is no command to erase this part of MRAM, so better generate and program `0xFF`s than scratch your head, what is wrong with the device/why it stopped working after moving to another board/

Reply:
I might have an idea, what's the case - did you try disabling MPI generation in recovery project?
Provisioning the device twice might be a strange idea :slightly_smiling_face:

Reply:
No, when I had storage.hex files for recovery and main application built separately - they erased the storage for each others.